pandas/core/generic.py

```diff
@@ -7825,6 +7828,49 @@ def _doc_parms(cls):
 
 """
 
+_any_also = """\
+See Also
+--------
+pandas.DataFrame.all : Return whether all elements are True \
+over requested axis."""
+
+_any_desc = """\
+Return whether any element is True over requested axis.
+
+One dimensional pandas.Series having boolean values will be returned. \
+Unlike pandas.DataFrame.all, pandas.DataFrame.any performs OR operation; \
+in other word, if any of the values along the specified axis is True, \
+pandas.DataFrame.any will return True."""
```

Comment:
Can you also mention here something that for Series the return value is a single boolean value?